Jellyfin.Server.Implementations/Migrations/20201004171403_MaxActiveSessions.cs

```diff
@@ -0,0 +1,27 @@
+﻿#pragma warning disable CS1591
+#pragma warning disable SA1601
+
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Jellyfin.Server.Implementations.Migrations
+{
+    public partial class MaxActiveSessions : Migration
```

Comment:
Should be `AddMaxActiveSessions` to be in line with the naming of other migrations

Reply:
```
-rw-r--r-- 1 joshua operator 1.7K Sep 24 21:15 20200905220533_FixDisplayPreferencesIndex.cs
-rw-r--r-- 1 joshua operator  16K Sep 24 21:15 20200905220533_FixDisplayPreferencesIndex.Designer.cs
```

This one does not use the format and was the one I based it on. I don't mind regenerating though.

Reply:
Reading comprehension fail again, "Add" vs "Fix" makes sense.

Reply:
Renamed and verified that this matches the expected values.